db/inmemory_test.go

```diff
@@ -0,0 +1,54 @@
+package db
+
+import "testing"
+
+func TestBasic(t *testing.T) {
+	im := NewInMemoryVolumeDatabase()
+
+	err := im.Connect()
+	if err != nil {
+		t.Fatal("Connect returned an error! " + err.Error())
+	}
+
+	list, err := im.List()
+	if err != nil {
+		t.Fatal("List returned an error! " + err.Error())
```

Comment:
I made the choice to make it fatal because if it fails, I cannot judge if the resulting calls worked. If we don't exit, we get no additional information as any further failures would not be meaningful.